internal/emlb/token-exchanger_test.go

```diff
@@ -53,12 +54,12 @@ func TestTokenExchanger_Token(t *testing.T) {
 	token, err := exchanger.Token()
 
 	// Assert that no error occurred
-	assert.NoError(t, err)
+	g.Expect(err).ToNot(HaveOccurred())
 
 	// Assert that the token is not nil
-	assert.NotNil(t, token)
+	g.Expect(token).ToNot(BeNil())
 
 	// Assert the token values
-	assert.Equal(t, "sample_token", token.AccessToken)
-	assert.Equal(t, time.Now().Add(time.Hour).Round(time.Second), token.Expiry.Round(time.Second))
+	g.Expect(token.AccessToken).To(Equal("sample_token"))
+	g.Expect(token.Expiry.Round(time.Second)).To(Equal(time.Now().Add(time.Hour).Round(time.Second)))
```

Comment:
Curious if rounding to minute would alleviate some flakes (wondering how many ticks happen between the token being generated and compared and whether IO (network) is involved)

Code is unchanged, so this is just a nitpic

Reply:
It's a unit test against a mock http server, so probably not a big risk, but still a good point.